developer/src/common/include/kmn_compiler_errors.h

```diff
@@ -41,19 +41,20 @@
 
 // Message codes
 //
-// All errors here and below are mirrored in kmc-kmn/src/compiler/messages.ts;
-// if you add a new error here be sure to update that file also. Note that this
-// correlation is currently maintained manually. All values must be below
-// 0x1000 (exclusive of severity code).
+// All errors here and below are mirrored in
+// kmc-kmn/src/compiler/kmn-compiler-messages.ts; if you add a new error here be
+// sure to update that file also. Note that this correlation is currently
+// maintained manually. All values must be below 0x1000 (exclusive of severity
+// code).
 #define CERR_None                                          0x00000000
 #define CERR_EndOfFile                                     0x00000001
 
 #define CERR_BadCallParams                                 0x00008002
 #define CERR_CannotAllocateMemory                          0x00008004
-#define CERR_InfileNotExist                                0x00008005
-#define CERR_CannotCreateOutfile                           0x00008006
+#define CERR_InfileNotExist                                0x00004005   // #10678: reduced from fatal to error in 17.0
```

Comment:
I assume these existed for the old compiler and were fatal then?  I feel like the comment doesn't make sense otherwise.

Reply:
They aborted the build but they are not a 'fatal' error in terms of how we define a fatal error now, which is an internal compiler error.